docs-content/index.html.md.erb

```diff
@@ -75,40 +74,122 @@ The following table provides version and version-support information about Cyber
++ **To configure the PCF tile, you must know the following information about your Conjur installation:**
+    <table class="nice">
+        <tr>
+            <td>
+                **Conjur Account**
+            </td>
+            <td>
+                Specified when Conjur is installed and configured. For hosted Conjur, the account is usually
+                an email address that you specify on the web form.
+            </td>
+        </tr>
+        <tr>
+            <td>
+                **URL of the Conjur instance**
+            </td>
+            <td>
+                May be the Conjur master load balancer if deploying in a High Availability (HA) configuration.
+            </td>
+        </tr>
+        <tr>
+            <td>
+                **Conjur Host Id for Service Broker**
+            </td>
+            <td>
+                Valid Conjur identity for a host that has `update` and `create` privileges on PCF-related
+                Conjur policy. The Service Broker uses these credentials to add and remove host identities
+                as you deploy and remove PCF applications. See [Recommendations](#recommendations).
+            </td>
+        </tr>
+        <tr>
+            <td>
+                **Conjur API key**
+            </td>
+            <td>
+                The Conjur API key associated with the **Conjur Host Id** for the service broker above.
+            </td>
+        </tr>
+        <tr>
+            <td>
+                **PCF Conjur Policy Branch ID**<br/>
+                (Recommended)
+            </td>
+            <td>
+                For production Conjur appliances, CyberArk strongly recommends a Conjur policy dedicated to PCF;
+                otherwise, the Conjur root policy is the default. See [Recommendations](#recommendations) for more information.
+            </td>
+        </tr>
+        <tr>
+            <td>
+                **URL of the Conjur follower load balancer**<br/>
+                (Conjur Enterprise v5.x+)<br/>
+                (Optional)
+            </td>
+            <td>
+                If deploying a cluster in an HA configuration, you can specify a URL for PCF apps to connect to
+                a Conjur follower to retrieve secret values rather than through the Conjur master. Cyberark strongly
+                recommends that you supply a load balanced follower address to support easier scalability.
+            </td>
+        </tr>
+        <tr>
+            <td>
+                **x509 CA Certificate Chain for Conjur**<br/>
+                (Conjur EE v5.x+ and Conjur OSS v1.x+)<br/>
+                (Optional)
+            </td>
+            <td>
+                The PEM encoded x509 certificate chain for the Conjur server. The certificate is
+                stored in a `.pem` file that was created by the `conjur init` command when using the
+                Conjur CLI. The default path for this file is `/conjur_install_directory/conjur-account.pem`.
```

Comment:
I think elsewhere in our docs we're advising users retrieve the cert using `openssl` - see [here](https://docs.conjur.org/Latest/en/Content/Integrations/jenkins-configure.htm?Highlight=PEM) for example. Ideally there would be a conjur docs page we could link to, but I couldn't find anything just now that just has generic information about retrieving your certificate chain.